### PR TITLE
Disable Experiments Option

### PR DESCRIPTION
**Pull request by calvinbui**
#### What this PR does / why we need it:

Gives the deployer the ability to disable deploying certain Chaos Experiments

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by calvinbui** [comment]
@ksatchit @ispeakc0de hey guys, calvin from the kubernetes slack. not sure how to assign reviews so i'm tagging you here.

File: charts/kubernetes-chaos/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.7.0"
 description: A Helm chart to install litmus chaos experiments for kubernetes category (chaos-chart)
 name: kubernetes-chaos
-version: 1.7.0
+version: 1.8.0
```
Comment (ispeakc0de) [inline charts/kubernetes-chaos/Chart.yaml:5]: Is this an auto-incremented version update? or you manually updated by you?

Comment (ksatchit) [inline charts/kubernetes-chaos/Chart.yaml:5]: Typically the app (litmus) release version and chart version coincide during the monthly release, with charts version getting bumped whenever we make changes until the next point/monthly release arrives. So, we could make this 1.7.1. WDYT @calvinbui ?

**Comment by ksatchit** [comment]
> @ksatchit @ispeakc0de hey guys, calvin from the kubernetes slack. not sure how to assign reviews so i'm tagging you here.

Hey @calvinbui !! Thank you for this PR -- this is a very good option for sure! Have a minor comment on the chart version. Otherwise LGTM!

File: charts/kubernetes-chaos/README.md [hunk 2]
```diff
@@ -2,11 +2,7 @@ kubernetes-chaos
 ================
 A Helm chart to install litmus chaos experiments for kubernetes category (chaos-chart)
 
-Current chart version is `1.6.2`
-
-
-
-
+Current chart version is `1.7.0`
```
Comment (ksatchit) [inline charts/kubernetes-chaos/README.md]: 👍

File: charts/kubernetes-chaos/Chart.yaml (hunk 1 above)
Comment (calvinbui) [reply charts/kubernetes-chaos/Chart.yaml:5]: Chart version was manually updated by me.

Comment (calvinbui) [reply charts/kubernetes-chaos/Chart.yaml:5]: Every chart I know of does this except for Elasticsearch

- https://github.com/helm/charts
- https://github.com/elastic/helm-charts

But at the end of the day it's your repo, I'm happy to follow whatever convention and it'll be worthwhile to document it as well.

**Comment by calvinbui** [comment]
@ksatchit any thoughts on the chart versioning?

**Review by ispeakc0de — APPROVED** [review]

**Review by gdsoumya — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Comment by ispeakc0de** [comment]
Thanks for the PR @calvinbui. Changes looks good :+1: :rocket:

**Pull request merged**